orchagent/aclorch.cpp

```diff
@@ -2378,7 +2382,11 @@ void AclOrch::queryAclActionCapability()
                 SWSS_LOG_WARN("Failed to query ACL %s action capabilities - "
                         "API assumed to be not implemented, using defaults",
                         stage_str);
-                initDefaultAclActionCapabilities(stage);
+                task_process_status handle_status = handleSaiGetStatus(SAI_API_SWITCH, status);
```

Comment:
>status [](http://example.com/codeflow?start=87&length=6)

If `SAI_STATUS_NOT_IMPLEMENTED == status`, the process will exit. But originally this is acceptable, and will init with default value.

Reply:
Agree, removed the handling for the ones with default values in failures.